class/minio.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
+  kapitan:
+    dependencies:
+      - type: helm
+        output_path: dependencies/minio/helmcharts/minio
+        source: https://charts.bitnami.com/bitnami
+        version: ${minio:charts:minio}
+        chart_name: minio
+    compile:
+      - input_paths:
+          - minio/component/app.jsonnet
+        input_type: jsonnet
+        output_path: apps/
+      - input_paths:
+          - minio/component/main.jsonnet
+        input_type: jsonnet
+        output_path: minio/
+      - output_path: minio/10_minio_helmchart
+        input_type: helm
+        output_type: yaml
+        input_paths:
+          - minio/helmcharts/minio
+        helm_params:
+          release_name: minio
+          namespace: "${minio/namespace}"
```

Comment:
Probably want to add

```
        helm_values:
          existingSecret: ${minio:adminCredentialsSecretName}        
```

and use `params.adminCredentials.secretName` when creating the credentials secret in `main.jsonnet`.

See comment on secret objects below

Reply:
Fixed. I guess you meant:
`existingSecret: ${minio:adminCredentials:secretName}` instead of `existingSecret: ${minio:adminCredentialsSecretName}` right?

Reply:
correct